tests/smoke/deploy.test.ts

```diff
@@ -71,9 +71,13 @@ describe('version check', () => {
       )
     },
   )
-  test('yarn monorepo multiple next versions site is compatible', { retry: 0 }, async () => {
-    await smokeTest(selfCleaningFixtureFactories.yarnMonorepoMultipleNextVersionsSiteCompatible)
-  })
+  test(
+    'yarn monorepo multiple next versions site is compatible',
+    { retry: 0, timeout: 1_000 * 60 * 5 },
```

Comment:
this change just bumps/sets timeout - we have 3 extensions being installed which we don't even use because site is in Netlify-testing account and that's pretty slow and this test case in particular was already slow before, but those 2 things combined meant that it was always timing out now